vscode-extension/package.json

```diff
@@ -3,7 +3,7 @@
     "displayName": "AL Object ID Ninja",
     "description": "Manage object IDs in multi-user environments with mind-boggling simplicity.",
     "publisher": "vjeko",
-    "version": "2.9.2",
```

Comment:
Please don't do this in future submissions. I'd like to control versioning of the extension myself. These kinds of changes are making my job more difficult.